packages/app-bridge/src/AppBridgeTheme.ts

```diff
@@ -98,13 +101,21 @@ export interface AppBridgeTheme<
         callback: EventCallbackParameter<EventName, Event>,
     ): EventUnsubscribeFunction;
 
+    /**
+     * @deprecated This will be removed in version 4.0.0 of `@frontify/app-bridge`
+     * Use `appBridge.context('portalId').get()` instead.
+     */
     getPortalId(): number;
 
+    /**
+     * @deprecated This will be removed in version 4.0.0 of `@frontify/app-bridge`
+     * Use `appBridge.context('brandId').get()` instead.
+     */
     getBrandId(): number;
 
     /**
-     * @deprecated This will be removed in version 4.0.0 of @frontify/app-bridge
-     * Use appBridge.dispatch(openNavigationManager()) for opening the Navigation Manager
+     * @deprecated This will be removed in version 4.0.0 of `@frontify/app-bridge`
+     * Use `appBridge.dispatch(openNavigationManager())` for opening the Navigation Manager
```

Comment:
nit (check other cases as well if you change this):
```suggestion
     * Use `appBridge.dispatch(openNavigationManager())` for opening the Navigation Manager.
```